### ISSUE TITLE
JSON body Pagination not firing

### ISSUE THREAD

**Issue by chriswhong** [qri-io/desktop#302]
The mechanism for requesting the next page of body data is only wired up for table views.

JSON view should behave similarly, requesting more data and refreshing the contents when the user scrolls to the bottom or top of the page.

**Comment by chriswhong** [comment]
No longer relevant after #313 (no more pagination on JSON bodies)
